### ISSUE TITLE
Maybe the effect of adapter is not such crucial?

### ISSUE THREAD

**Issue by Rainlt** [GenjiB/LAVISH#11]
I have tried to delete the adapter in both AVE and AVQA task, without other config changed. And the performance only decreased for around 1~2%. If we unlock the layernorm in original swinv2, the gap will be further narrowed. But if I finetune all parameters (without adapter) in AVE, the performance decreased a lot, maybe the learning schedule needs to be adjusted simultaneously.

And there is an interesting phenomenon that if I delete the adapter, the model need more epochs to achieve the best performance, which indicated that the added trainable parameters can accelerate the training.  Of course, it may also be caused by the need to adjust the learning rate.

**Comment by GenjiB** [comment]
Do you also tune any layers of Swin-V2 Based on my experience, if you remove LAVISHs (all frozen), the performance will drop ~ 4% like Table 5. 

Sure, if you tune layernorm only, it calls BitFit. It is also a parameter-efficient fine-tuning approach.  You can check their paper if you are interested in.

Also, if you fine-tune the Swin-V2, I also find the performance will drop. Since large pretrained models will overfit AVE dataset (~4K videos), adding adapter would be a way to alleviate this issue/

**Comment by Rainlt** [comment]
> Do you also tune any layers of Swin-V2 Based on my experience, if you remove LAVISHs (all frozen), the performance will drop ~ 4% like Table 5.
> 
> Sure, if you tune layernorm only, it calls BitFit. It is also a parameter-efficient fine-tuning approach. You can check their paper if you are interested in.
> 
> Also, if you fine-tune the Swin-V2, I also find the performance will drop. Since large pretrained models will overfit AVE dataset (~4K videos), adding adapter would be a way to alleviate this issue/

Thank you for your kind response. I can get the identical result (77.94 in epoch 18) for swin-v2 without lavish as shown in table 5, but I can only achieve 78.73 in epoch 4 with lavish. Maybe there are still some gaps? I‘ll check the code later.
